src/package.json

```diff
@@ -5,6 +5,8 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.28",
     "@fortawesome/free-solid-svg-icons": "^5.13.0",
     "@fortawesome/react-fontawesome": "^0.1.9",
+    "lodash.isequal": "^4.5.0",
+    "lodash.xorwith": "^4.5.0",
```

Comment:
I'd also be OK with including all of `lodash`, since I happen to like it and we have no need to economize on bundle size.

Reply:
Sure, I'll get to this.